homeassistant/components/flo/device_action.py

```diff
@@ -0,0 +1,106 @@
+"""Provides device actions for Flo by Moen devices."""
+from typing import List
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_DEVICE_ID, CONF_DOMAIN, CONF_TYPE
+from homeassistant.core import Context, HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType, TemplateVarsType
+
+from .const import DOMAIN as FLO_DOMAIN
+from .device import FloDeviceDataUpdateCoordinator
+from .services import (
+    ATTR_DEVICE_ID,
+    ATTR_LOCATION_ID,
+    ATTR_REVERT_TO_MODE,
+    ATTR_SLEEP_MINUTES,
+    SERVICE_RUN_HEALTH_TEST,
+    SERVICE_SET_AWAY_MODE,
+    SERVICE_SET_HOME_MODE,
+    SERVICE_SET_SLEEP_MODE,
+    SYSTEM_MODE_HOME,
+)
+
+FLO_ACTION_TYPE_SERVICE_CALL = "service_call"
+
+ACTION_SCHEMA = cv.DEVICE_ACTION_BASE_SCHEMA.extend(
+    {vol.Required(CONF_DOMAIN): FLO_DOMAIN, vol.Required(CONF_TYPE): str}
+)
+
+DEVICE_ACTIONS = [
+    {CONF_TYPE: SERVICE_RUN_HEALTH_TEST, CONF_DOMAIN: FLO_DOMAIN},
```

Comment:
Is this important enough for easy mode that it should show up in device actions ? Or is it ok if we leave it as just a service ?

Reply:
I think device actions are easier to use, I’d prefer to keep the actions and de-register the services if given the option. If we did that, we could ditch the device state attributes questioned below as well...

Reply:
We can't. All device triggers/conditions/actions need to use things that one can achieve with YAML too.

Reply:
Oh, I think I missed the thinking in the original question. Were you only questioning the health test action and not the device actions in general? If so, I can remove the health test action if you really think having it would confuse easy mode folks.

Reply:
Yeah the health.

Reply:
removed all actions since the entity services aren't really that different